crates/sdk/src/network-v2/types.rs

```diff
@@ -0,0 +1,87 @@
+use std::fmt;
+
+/// The default URL for the proof network explorer.
+pub const DEFAULT_EXPLORER_URL: &str = "https://network.succinct.xyz";
+
+/// A 32-byte hash that uniquely identifies a proof request.
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct RequestId(Vec<u8>);
```

Comment:
RequestHash, to be consistent?

Reply:
It's request_id everywhere in the proto, on the RPC, and on the frontend - to avoid a massive refactor I feel like leaving at RequestId is fine